pkg/account/downloader/internal/http/client.go

```diff
@@ -41,6 +41,29 @@ func (c *Client) GetUsers(ctx context.Context, accountUUID string) ([]accountman
 	return r.Items, nil
 }
 
+func (c *Client) GetServiceUsers(ctx context.Context, accountUUID string) ([]accountmanagement.ExternalServiceUserDto, error) {
+	serviceUsers := []accountmanagement.ExternalServiceUserDto{}
+	const pageSize = 10
+	for page := 1; ; page++ {
+		r, resp, err := c.ServiceUserManagementAPI.GetServiceUsersFromAccount(ctx, accountUUID).Page(float32(page)).PageSize(pageSize).Execute()
+		defer closeResponseBody(resp)
```

Comment:
Using defer in a loop will release the resources at the end of all iterations of the loop. I suggest moving this code into a separate method, calling it in a loop, and deferring in the method. In this way, the release of resources can happen earlier.